docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,283 @@
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra),
+- contributing to the maintenance of Gammapy products on open-source repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers (e.g module maintainer),
+- organizing and leading hands-on sessions or schools,
+- managing our social media accounts,
+- and coordinating the project by the Lead Developers.
+
+Any contribution without threshold on these topics renders a user as official Gammapy contributor. Many of these contributions
+are tracked using GitHub. However, the use of personal data is regulated in Europe and one should have the
+authorisation of users to be cited with their personal information (typically, their affiliation and their ORCID if any).
+For this reason, we propose to use the GitHub `Developer Certificate of Origin <https://github.com/apps/dco>`_ (DCO) on
+Pull Request. Its acceptation by contributors will permit to:
+
+- certify that a user wrote or otherwise has the right to submit a code under our open source licence,
+- allow our project to use their personal data for the contributions' record (ie for releases).
+
+The file ``README.md`` will contain a `Contributing` section explaining that the project follows the GitHub DCO, as well
+as a link to this PIG.
+
+.. important::
+    We strongly recommend that the contributors use always the same GitHub account with a valid full name, to
+    synchronise their git email addresses with their GitHub addresses, to have a unique `ORCID <https://orcid.org/>`_
+    identifier and store it into their GitHub user profile for commodity, and inform the developers of any change of
+    affiliation.
+
+
+Definition of the `Maintainer`
+------------------------------
+
+Usually, a software maintainer or package maintainer is one or more people who build source code into a binary package
+for distribution, commit patches, or organize code in a source repository. In the Open Source community, they are
+generally mentioned in the metadata associated with a release.
+
+For Gammapy, by default, the maintainers are the Lead Developers. If in the future a task is dedicated to the creation
+of a release, the maintainer will be the person in charge of this task.
+
+Releases
+--------
+
+Each Gammapy release should be associated with an updated list of authors that will be public on repository (Zenodo and
+HAL/SW). This section is about feature releases. Bug releases will have as list of authors the one of its associated
+feature release whose rules are described here, plus the potentiel additional contributors.
+
+The list of authors is composed of people who contributed to the current release (see :ref:`_contributor`) and who have
+accepted the GitHub DCO. If the DCO is not accepted by a contributor, her/his contribution might not be usable and thus
```

Comment:
The "Gammapy Contribution Terms" or similar, the actual content should be defined in this PIG.

The DCO comes from the Linux and Git projects, not Github. The app you link is a thirdparty app, also not from Github.

Using signed-off-by does not relate directly to the specific DCO of Linux and Git, it is defined for each project individually, see the git help (`man git commit`):

> Add a Signed-off-by trailer by the committer at the end of the commit log message. The meaning of a signoff depends on the
> project to which you’re committing. For example, it may certify that the committer has the rights to submit the work under the
> project’s license or agrees to some contributor representation, such as a Developer Certificate of Origin. (See
> http://developercertificate.org for the one used by the Linux kernel and Git projects.) Consult the documentation or leadership
> of the project to which you’re contributing to understand how the signoffs are used in that project.


It mentions the git and linux DCOs as one possibibility, but projects have to define their own policy. We could define that we use the same or make our own with additional or fewer terms, but we need to define it (here and also I think in the Pull Request template and `CONTRIBUTING.md` and the developer docs).


I think the Linux and Git DCO is a good starting point, but we could add things specific to gammapy, e.g. mention the usage of the PII for publication author lists.

Reply:
Corrections and addition made